### PR TITLE
Tt factory bot faker setup

### PR DESCRIPTION
**Pull request by tylertomlinson**
**ALWAYS PULL FROM MASTER BEFORE CREATING PR**

### All Submissions:

* [•] Have you followed the guidelines in our Contributing document? (need to decided as group what to put here)
* [•] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls)? (for reducing merge conflicts)

### New Feature Submissions:

1. [•] Does your submission pass tests?
2. [•] Have you lint your code locally prior to submission? 

### Changes to Core Features:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

Added `factory bot` and `faker` gem files

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -13,9 +13,10 @@ gem 'sass-rails', '~> 5.0'
 gem 'uglifier', '>= 1.3.0'
 gem 'coffee-rails', '~> 4.2'
 gem 'jbuilder', '~> 2.5'
+gem 'factory_bot_rails'
```
Comment (alex-latham) [inline Gemfile:16]: Usage documentation: https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md

File: Gemfile [hunk 2]
```diff
@@ -13,9 +13,10 @@ gem 'sass-rails', '~> 5.0'
...
+gem 'faker'
```
Comment (alex-latham) [inline Gemfile:17]: Usage documentation: https://github.com/faker-ruby/faker#usage

**Review by alex-latham — APPROVED** [review]
This looks good to me

**Review by danielpselph — APPROVED** [review]
Need to make sure to reference docs when implementing

**Pull request merged**